KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/TikaTextExtractor.java

```diff
@@ -79,6 +89,42 @@ public Reader getReader(Content content) throws TextExtractorException {
         officeParserConfig.setUseSAXPptxExtractor(true);
         officeParserConfig.setUseSAXDocxExtractor(true);
         parseContext.set(OfficeParserConfig.class, officeParserConfig);
+        
+        // configure OCR if it is enabled in KWS settings and installed on the machine
+        if (TESSERACT_PATH != null && KeywordSearchSettings.getOcrOption()) {
+            
+            // configure PDFParser. 
```

Comment:
It appears there's a lot of commentary here. Is this copy/paste info? I see lines of code that are commented out. It would be a little easier to follow the code with fewer comments.

Reply:
Done.